tests/test_plugins/pytester_example_dir/test_file_3.py

```diff
@@ -0,0 +1,18 @@
+""" This is a test file used for testing the pytest plugin. """
+from pint.unit import Unit
+
+
+def test_complex_snapshot(snapshot):
+    """ The snapshot for this function is expected to exist, but the assertion fails. """
+
+    class HasUnitsClass:
+        def __init__(self):
+            tmp_dict = {"unit": Unit("m"), "value": -1000}
+            tmp_list = [tmp_dict, tmp_dict]
+
+            self.test1 = tmp_dict
+            self.test2 = tmp_list
+
+    klass = HasUnitsClass()
```

Comment:
(typo?) klass -> test_class

Reply:
[Fixed in this commit](https://github.com/MORSECorp/snappiershot/pull/96/commits/38efbcfc36b68c785dfb55cc40eeb3ba35efcef5)